src/components/user/tour/tour.js

```diff
@@ -1,6 +1,67 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import { DataGrid } from '@material-ui/data-grid';
 
-export default function Tour() {
-  return <Typography paragraph>The 'Tour' Page comming soon...</Typography>;
+const rows = [
+  {
+    id: 1, 
+    status: 'All booked',
+    name: 'Bellinis in Armenia',
+    origin: 'Italy',
+    days: '7',
+    date: 'Jan 28-Feb 11',
+    destination: 'Yerevan, Sevan, Dilijan',
+    guests: 'Marco Belini, Chiara Belini',
+    hotels: 'Noy Land, Popock 1',
+    activities: 'Yell Park, Papanino',
+    flight: 'QR 706',
+    price: '$5400',
+    notes: 'Very picky person',
+    createdBy: 'Armine G',
+   },
+   {
+    id: 1, 
+    status: 'All booked',
+    name: 'Bellinis in Armenia',
+    origin: 'Italy',
+    days: '7',
+    date: 'Jan 28-Feb 11',
+    destination: 'Yerevan, Sevan, Dilijan',
+    guests: 'Marco Belini, Chiara Belini',
+    hotels: 'Noy Land, Popock 1',
+    activities: 'Yell Park, Papanino',
+    flight: 'QR 706',
+    price: '$5400',
+    notes: 'Very picky person',
+    createdBy: 'Armine G',
+   },
+];
+
+const columns = [
+  { field: 'status', headerName: 'Status'},
+  { field: 'name', headerName: 'Name', width: 150 },
+  { field: 'origin', headerName: 'Origin' },
+  { field: 'days', headerName: 'Nr. of days' },
+  { field: 'destination', headerName: 'Destination', width: 150 },
+  { field: 'date', headerName: 'Dates', width: 150 },
+  { field: 'guests', headerName: 'Guests', width: 150 },
+  { field: 'hotels', headerName: 'Hotels', width: 150 },
+  { field: 'activities', headerName: 'Activities', width: 150 },
+  { field: 'flight', headerName: 'Flight' },
+  { field: 'price', headerName: 'Price' },
+  { field: 'notes', headerName: 'Notes' },
+  { field: 'createdBy', headerName: 'Created By' },
+];
+
+export default class Tour extends React.Component {
+  constructor(props) {
```

Comment:
There is no need to add the constructor function if you don't use props or state within it. 
In the terminal, you'll message like this: 
`src/components/user/tour/tour.js
  Line 56:3:  Useless constructor  no-useless-constructor
`

If the component has only the `render()` method, no need to make it a class component, just use a functional component.

Reply:
I'm going to load data in componentWillUnmount action
is it good idea?

Reply:
`componentWillUnmount` ?? Never load data in `componentWillUnmount`.
`componentDidMount` method is the best place to load the data.

Reply:
please check the documentation` [https://reactjs.org/docs/react-component.html#componentdidmount](https://reactjs.org/docs/react-component.html#componentdidmount).